### PR TITLE
Bump versions for Iron Irwini Patch 7

### PR DESCRIPTION
**Pull request by Yadunund**
Final version bumps before marking Iron EOL.

Once this PR is approved, I will run CI and packaging jobs resp.

### CODE REVIEW

**Review by clalancette — APPROVED** [review]
I've left one comment inline; I think we should do a new RViz release, since there has been an update there.  But it isn't super important, so I'm going to approve anyway and leave it to you whether you want to do that release or not.

**Comment by Yadunund** [comment]
CI started with: https://raw.githubusercontent.com/ros2/ros2/refs/heads/yadu/iron-patch-7/ros2.repos
* Linux [Build Status](http://ci.ros2.org/job/ci_linux/22185/)
* Linux-aarch64 [Build Status](http://ci.ros2.org/job/ci_linux-aarch64/16437/)
* Linux-rhel [Build Status](http://ci.ros2.org/job/ci_linux-rhel/1706/)
* Windows [Build Status](http://ci.ros2.org/job/ci_windows/22934/)

File: ros2.repos [hunk 1]
```diff
@@ -382,7 +382,7 @@ repositories:
   ros2/rviz:
     type: git
     url: https://github.com/ros2/rviz.git
-    version: 12.4.8
+    version: 12.4.9
```
[the remark this answers is not in the document]
Comment (Yadunund) [reply ros2.repos]: Bloomed a new release with the changes https://github.com/ros/rosdistro/pull/43706

**Comment by Yadunund** [comment]
Packaging jobs started with: https://raw.githubusercontent.com/ros2/ros2/refs/heads/yadu/iron-patch-7/ros2.repos

* Linux [Build Status](https://ci.ros2.org/view/packaging/job/packaging_linux/3645/)
* Linux-aarch64 [Build Status](https://ci.ros2.org/view/packaging/job/packaging_linux-aarch64/2992/)
* Linux-rhel [Build Status](https://ci.ros2.org/view/packaging/job/packaging_linux-rhel/2144/)
* Windows [Build Status](https://ci.ros2.org/view/packaging/job/packaging_windows/3445/)
* Windows debug [Build Status](https://ci.ros2.org/view/packaging/job/packaging_windows_debug/2411/)

**Pull request merged**